asterius/test/endianness/endianness.hs

```diff
@@ -0,0 +1,39 @@
+
+-- import Network.Socket -- Well, it's not booted, this one, so we cannot test it :(
```

Comment:
See https://github.com/tweag/asterius/blob/master/.github/workflows/pipeline.yml#L86, `network` is actually booted in CI.

Reply:
It is? Locally I could not find it. Maybe `./utils/reboot.sh` is out-of-sync.

Reply:
> See https://github.com/tweag/asterius/blob/master/.github/workflows/pipeline.yml#L86, `network` is actually booted in CI.

Indeed, running

> . ./.envrc
> ahc-cabal v1-update 
> pushd ghc-toolkit/boot-libs
> ahc-cabal v1-install -j2 wai-middleware-caching
> popd

locally worked just fine. I'll add a test for `Network.Socket` as well, to make sure #756 is addressed :+1: